client/app/bundles/course/duplication/pages/Duplication/ItemsSelector/VideosSelector.jsx

```diff
@@ -57,24 +57,36 @@ class VideosSelector extends React.Component {
     );
   }
 
-  render() {
+  renderBody() {
     const { videos } = this.props;
-    if (!videos) { return null; }
+
+    if (videos.length < 1) {
+      return (
+        <Subheader>
+          <FormattedMessage {...translations.noItems} />
+        </Subheader>
+      );
+    }
 
     return (
       <div>
-        <h2><FormattedMessage {...defaultComponentTitles.course_videos_component} /></h2>
         <BulkSelectors
           callback={this.setAllVideoSelection}
           styles={{ selectLink: { marginLeft: 0 } }}
         />
-        {
-          videos.length > 0 ?
-          videos.map(video => this.renderRow(video)) :
-          <Subheader>
-            <FormattedMessage {...translations.noItems} />
-          </Subheader>
-        }
+        { videos.map(video => this.renderRow(video)) }
+      </div>
+    );
+  }
+
+  render() {
+    const { videos } = this.props;
+    if (!videos) { return null; }
```

Comment:
Can I suggest instead that bulk selectors be rendered only if videos.length is 2 or greater? Doesn't make sense to have bulk selector for 1 video only.